src/patternfly/components/Tabs/examples/Tabs.md

```diff
@@ -92,24 +92,24 @@ import './Tabs.css'
 {{/tabs}}
 ```
 
-### Vertical expandable example (legacy)
-```hbs
+### Vertical expandable example (deprecated)
```

Comment:
We probably don't need the `(deprecated)` anymore with the new label being applied.

Could we also add the deprecated label to the Card examples marked deprecated?

Reply:
for now I left it because there were other examples with the same name without it. We can do a follow up in the future to rename these examples in a better way.